code/game/gamemodes/cult/cult.dm

```diff
@@ -120,7 +125,17 @@
 
 	var/datum/antagonist/cult/new_cultist = new()
 	new_cultist.give_equipment = equip
+	if(cult_mind.add_antag_datum(new_cultist,cult_team))
+		if(stun)
+			cult_mind.current.Unconscious(100)
+		return TRUE
 
+/datum/game_mode/proc/add_cultist_golem(datum/mind/cult_mind, stun , equip = FALSE, datum/team/cult/cult_team = null)
+	if (!istype(cult_mind))
+		return FALSE
+
+	var/datum/antagonist/cult/golem/new_cultist = new()
+	new_cultist.give_equipment = equip
```

Comment:
make a var on the orginal proc (golem = FALSE) instead of just making another identical proc